diagnostic.md

```diff
@@ -21,13 +21,16 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --include --request GET "http://www.example.com"
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts =$.ajax({
+  url: "http://www.example.com"
+  method: "GET",
+});
 ```
```

Comment:
this is technically right but you don't want to just store the request in a variable.  You want to wrap it in a function.